SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -279,7 +279,11 @@ bool USpatialActorChannel::ReplicateActor()
 		const int32 HistoryIndex = i % FRepChangelistState::MAX_CHANGE_HISTORY;
 		FRepChangedHistory& HistoryItem = ChangelistState->ChangeHistory[HistoryIndex];
```

Comment:
This code looks very similar to the one below, could we refactor this out to a function?

Reply:
Probably out of the scope of this PR. This is a quick bugfix for Scavengers.

Reply:
Could you create a ticket? I feel like it wouldn't take long to just touch this up though

Reply:
Created ticket: https://improbableio.atlassian.net/browse/UNR-663